packages/headless/src/features/insight-search/insight-search-slice.test.ts

```diff
@@ -0,0 +1,224 @@
+import {logSearchboxSubmit} from '../query/query-analytics-actions';
```

Comment:
We should not need this test file anymore since we're reusing the `searchReducer`.

Reply:
good catch